integration-tests/src/test/kotlin/org/stellar/anchor/platform/PlatformApiTests.kt

```diff
@@ -0,0 +1,221 @@
+package org.stellar.anchor.platform
+
+import java.time.temporal.ChronoUnit.SECONDS
+import org.junit.jupiter.api.Assertions.*
+import org.stellar.anchor.api.platform.PatchTransactionRequest
+import org.stellar.anchor.api.platform.PatchTransactionsRequest
+import org.stellar.anchor.api.sep.sep12.Sep12PutCustomerRequest
+import org.stellar.anchor.api.sep.sep12.Sep12Status
+import org.stellar.anchor.api.sep.sep31.Sep31PostTransactionRequest
+import org.stellar.anchor.api.shared.Amount
+import org.stellar.anchor.auth.AuthHelper
+import org.stellar.anchor.auth.JwtService
+import org.stellar.anchor.event.models.TransactionEvent
+import org.stellar.anchor.reference.client.PlatformApiClient
+import org.stellar.anchor.util.GsonUtils
+
+lateinit var platformApiClient: PlatformApiClient
+
+class PlatformApiTests {
+  companion object {
+    fun setup() {
+      if (!::platformApiClient.isInitialized) {
+        val platformToAnchorJwtService = JwtService("myAnchorToPlatformSecret")
+        val authHelper =
+          AuthHelper.forJwtToken(platformToAnchorJwtService, 900000, "http://localhost:8081")
+        platformApiClient = PlatformApiClient(authHelper, "http://localhost:8080")
+      }
+    }
+
+    fun `test sep24 doposit, widhdraw, get and patch`() {}
```

Comment:
empty test, should delete or add TODO for implementation

Reply:
// TODO: added